### DISCUSSION TITLE
Would gpu processing be possible in the future?

### DISCUSSION THREAD

**Discussion by chris-sanders** [sn4k3/UVtools#596] [Ideas]
While I've done development I haven't worked with gpu's myself. I have a Saturn 2, I tend to print at 30um and files are rather slow to fully process. I notice the operations you're doing seem to fit the ideal scenario for GPGPU use cases. I say that mainly considering the pixel processing and number of layers to process most of which appears like it could be processed with a very high amount of parallelism.

Have you considered or do you think in the future optionally using GPU to accelerate at least some of the detection and remediation would be on the road map? Sorry if you've seen this before I couldn't find anything with gpu in it, or the search in discussion is poor :)

**Comment by sn4k3** [comment]
The results are worse with GPU in my past attempts.
Read here: https://github.com/sn4k3/UVtools/issues/281
